pkg/rp/v1/outputresource.go

```diff
@@ -27,6 +27,16 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// DeploymentStatus - Status of Output Resource deployment
```

Comment:
We need a clearer comment here. What is an output resource deployment for a contributor who is not very familiar with the code?